common/dynamicconfig/constants.go

```diff
@@ -2429,6 +2429,12 @@ that task will be sent to DLQ.`,
 		`SendRawHistoryBetweenInternalServices is whether to send raw history events between internal temporal services`,
 	)
 
+	EnableRequestIdRefLinks = NewGlobalBoolSetting(
```

Comment:
Add some comment that this is a temporary config that will be removed in the next release.